conda/recipes/cucim/meta.yaml

```
@@ -59,7 +59,7 @@ requirements:
     - rapids-build-backend >=0.4.0,<0.5.0
     - scikit-image >=0.19.0,<0.25.0
     - scipy >=1.6
-    - setuptools >=80.9.0
+    - setuptools>=80.9.0
```

```suggestion
    - setuptools >=80.9.0
```

reverting this so this PR does not change this line